### DISCUSSION TITLE
Addon: Building a Custom Panel

### DISCUSSION THREAD

**Discussion by AS-EBaiden** [storybookjs/storybook#26235] [Help]
### Summary

I've been working on setting up a custom Notes panel in my project, managed through **manager.js.** While I've successfully configured the panel, I'm encountering difficulties accessing specific data I intend to display.

The challenge lies in retrieving the **customNotes** passed within the Meta parameters, as illustrated in this screenshot: 

![image](https://github.com/storybookjs/storybook/assets/110992392/12394ee4-456e-4ad1-9944-6474e88fedca)

In my **manager.js** file, I've made necessary configurations as depicted here: 
![image](https://github.com/storybookjs/storybook/assets/110992392/b91db547-df25-49cf-b43e-fac67f8c0017)

Upon logging the API, I observed a potentially useful structure, shown in this image: 
![image](https://github.com/storybookjs/storybook/assets/110992392/85d15797-aba3-4c63-8853-7e820df958e7)

However, I require assistance in pinpointing where I can locate this custom attribute for various components.

Any guidance on accessing and utilizing this **customNotes** attribute within different components would be greatly appreciated. Thank you in advance for your help!

### Additional information

this is my manager.js

```
import React from 'react'
import { addons, types } from '@storybook/addons'
// import { create } from '@storybook/theming/create'

import { MDXProvider } from '@mdx-js/react'

/**
 * Themes are messed up in storybook@6.5.0-alpha.51
 */
// addons.setConfig({
//   theme: create({
//     base: 'light',
//     fontCode: 'monospace',
//   }),
// })



addons.register('myaddon', (api) => {
  console.log('APEE', api)
  addons.add('myaddon/panel', {
    type: types.PANEL,
    title: 'MY Addon',
    render: <div>Hello there </div>,
  })
})
```

Also my current Storybook version is 7.6.17

**Comment by Marklb** [comment]
You could probably access it from the api, but maybe something like the following would work better? *I don't deal with addons much, so I don't have many examples at the moment,*

```js
import React from 'react';
import { addons, types, useParameter } from '@storybook/manager-api';

addons.register('myaddon', (api) => {
  console.log('APEE', api);

  addons.add('myaddon/panel', {
    type: types.PANEL,
    title: 'MY Addon',
    render: () => {
      const parameters = useParameter();
      return <div>Hello theret </div>;
    },
  });
});
```
